app/modules/backup/continuous-replication/index.js

```diff
@@ -100,10 +101,11 @@ export default angular.module('backup.continuousReplication', [
       this.formData.tag = tag
       this.formData.selectedSr = selectedSr
       this.formData.scheduleId = schedule.id
+      this.formData.reportWhen = reportWhen
       this.scheduleApi.setCron(cronPattern)
     }
 
-    this.save = (id, vms, tag, sr, cron, enabled) => {
+    this.save = (id, vms, tag, sr, cron, enabled, reportWhen) => {
```

Comment:
Please clarify. How is the new "reportWhen" implementation any different from the preexisting options such as tag?